deploy/olm-catalog/argocd-operator/0.7.0/argocd-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -1,16 +1,17 @@
 apiVersion: v1
+apiVersion: v1
 kind: Service
 metadata:
   creationTimestamp: null
   labels:
-    control-plane: argocd-operator
+    control-plane: controller-manager
   name: argocd-operator-controller-manager-metrics-service
 spec:
   ports:
   - name: https
     port: 8443
     targetPort: https
   selector:
-    control-plane: argocd-operator
+    control-plane: controller-manager
```

Comment:
@svghadi is this an intentional change?

Reply:
I cherry-picked the changes from https://github.com/argoproj-labs/argocd-operator/pull/966. 

According to https://github.com/argoproj-labs/argocd-operator/pull/966#issuecomment-1685690158 looks like a intentional change.